app/assets/stylesheets/category.css

```diff
@@ -0,0 +1,92 @@
+.title {
+  background-color: #5fb523;
+  padding: 1rem;
+  text-align: center;
+  color: white;
+  position: relative;
+}
+
+#categories {
```

Comment:
- [ ] It is not recommended to use ID as a selector. You can use classes instead.

Reply:
Implemented thank you for reviewing. :+1: